core/tests/protractor_utils/CreatorDashboardPage.js

```diff
@@ -91,6 +93,14 @@ var CreatorDashboardPage = function() {
     waitFor.pageToFullyLoad();
   };
 
+  this.clickCreateExplorationButtonOnMobile = function() {
```

Comment:
Is this "Create" button not clickable/visible on Desktop?

Reply:
I think you meant Mobile, instead of Desktop? There is no "Create" button in mobile.

Reply:
I did a quick check. Looks like this button shows up on both Desktop and Mobile. However, it only appears when the /creator_dashboard is completely empty, i.e when a fresh user log-in and navigate to /creator_dashboard

Reply:
I think that button should be named `protractor-test-create-new-exploration-button` since there is already a `protractor-test-create-activity-button` up in the navbar

Reply:
The __navbar__'s "Create" button will not be visible. It is already tagged.
You are tagging the 2nd __Create Exploration__ button, which is visible on both Desktop and Mobile. To my point, if it is visible/ clickable on both versions, it should not be named `protractor-test-mobile...`since it implies it is only for mobile version.

Reply:
Oh, cool! Got it. But wasn't your original query about the "Create" button [here](https://github.com/oppia/oppia/pull/5340#discussion_r205893727)? Therefore, I answered keeping that in mind :)

Reply:
Sooo sorry for creating confusion there... I should stop talking about multiple things at once. My bad! No problem!

Reply:
Done, thanks!